packages/manager/src/features/components/PlansPanel/PlanSelectionTable.tsx

```diff
@@ -97,19 +133,24 @@ export const PlanSelectionTable = (props: PlanSelectionTableProps) => {
                 {isPlanCell && filterOptions?.header
                   ? filterOptions?.header
                   : cellName}
-                {showTransferTooltip(cellName) && (
-                  <TooltipIcon
-                    sxTooltipIcon={{
-                      height: 12,
-                      marginTop: '-2px',
-                      ml: 0.5,
-                      px: 0,
-                      py: 0,
-                    }}
-                    status="help"
-                    text="Some plans do not include bundled network transfer. If the transfer allotment is 0, all outbound network transfer is subject to standard charges."
-                  />
-                )}
+                {showTransferTooltip(cellName) &&
+                  showTooltip(
+                    'help',
+                    'Some plans do not include bundled network transfer. If the transfer allotment is 0, all outbound network transfer is subject to standard charges.'
+                  )}
+                {showUsableStorageTooltip(cellName) &&
+                  showTooltip(
+                    'help',
+                    <Typography>
+                      Usable storage is smaller than the actual plan storage due
+                      to the overhead from the database platform.{' '}
+                      <Link to="https://www.linode.com/pricing/">
+                        Learn more
+                      </Link>
+                      .
+                    </Typography>,
```

Comment:
After talking with UX about our latest patterns, Matthew has confirmed that no new tooltips should contain "Learn more" links, so I'm requesting that small change here. There may be some existing tooltips where the link exists, but let's not introduce it here.

```suggestion
                    'Usable storage is smaller than the actual plan storage due
                      to the overhead from the database platform.',
```

Reply:
I've removed link, thanks!